Emby.Server.Implementations/Library/PathExtensions.cs

```diff
@@ -120,12 +105,86 @@ public static class PathExtensions
                 return false;
             }
 
-            var newSubPathTrimmed = newSubPath.AsSpan().TrimEnd(newDirectorySeparatorChar);
+            var newSubPathTrimmed = newSubPath.AsSpan().TrimEnd((char)newDirectorySeparatorChar!);
```

Comment:
```suggestion
            var newSubPathTrimmed = newSubPath.AsSpan().TrimEnd(newDirectorySeparatorChar);
```

This change isn't required for me